src/LineBlockTracer.h

```diff
@@ -33,6 +33,14 @@ class cLineBlockTracer :
 	typedef cBlockTracer super;
 
 public:
+	enum eLineOfSight
```

Comment:
These three are the most common LineOfSight options. If a plugin wants a different one, it will need to use the full `cLineBlockTracer:Trace()` method with custom callbacks.

Even like this it is an improvement over the old `cTracer` that had only one option for LineOfSight tracing, equivalent to `losAirWater`.